CHANGELOG.md

```diff
@@ -4,3 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 ### Added
+
+- The primary key(s) is / are now appended to the to the list of fields used
```

Comment:
Could you shorten this? The change-log has to be clean and concise, focused on the action.

Something like: `Added default sort for primary key(s)`.